src/components/AppMap.ts

```diff
@@ -834,9 +836,50 @@ export default class AppMap extends mixins(MixinUtil) {
       marker.data.getMarker().addTo(this.map.m);
     }
 
+    this.updateTooltip();
     this.searching = false;
   }
 
+  yTooltipOnEach(marker: any) {
```

Comment:
enableYTooltip (or something else that makes it clearer what this function does)